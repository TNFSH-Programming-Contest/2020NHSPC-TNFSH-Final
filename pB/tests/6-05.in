4
-993.718 -308.245
-993.718 -343.244
359.335 -343.244
359.335 -308.245
-107.415 -324.032
