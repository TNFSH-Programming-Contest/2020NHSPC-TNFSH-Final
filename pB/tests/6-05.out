2468984.291
